lib/create-matcher/matcher-prototype.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+var matcherPrototype = {
+    toString: function() {
+        return this.message;
+    }
+};
+
+matcherPrototype.or = function(valueOrMatcher) {
+    var createMatcher = require("../create-matcher");
```

Comment:
Alternatively, you could declare `var createMatcher;` on the top level and assign `createMatcher = require("../create-matcher");` below the `module.exports` section.

Reply:
That's interesting. I couldn't get that to work.

I'll merge this PR, then we can make that improvement work later.